mlflow/gateway/constants.py

```diff
@@ -7,3 +7,6 @@
 # Provider constants
 MLFLOW_AI_GATEWAY_ANTHROPIC_MAXIMUM_MAX_TOKENS = 1_000_000
 MLFLOW_AI_GATEWAY_ANTHROPIC_DEFAULT_MAX_TOKENS = 200_000
+
+# Disallowed payload keys
```

Comment:
this is unused in this PR. 
If we need to check for additional forbidden fields in the future, we can increase the complexity with an exclusion set and an intersection to check for forbidden keys. Since `model` is all we care about with the current provider APIs, I'm deleting this unused constant.